src/kokab/utils/common.py

```
@@ -3,15 +3,10 @@
 
 
 import json
-from typing import Dict, List, Tuple, Union
+from typing import Dict, List, Tuple
```

Ruff (0.14.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)